### ISSUE TITLE
Scheduler events with allDayEventTemplate overlap in the allDaySlot

### ISSUE THREAD

**Issue by IvanDanchev** [telerik/kendo-ui-core#2985]
### Bug report
If the allDayEventTemplate option is set the events in the allDaySlot overlap.

### Reproduction of the problem
[Dojo](https://dojo.telerik.com/@Dimitar-Goshev/ecajatay) example.
[Screenshot](https://www.screencast.com/t/Q1pKqjrgc). Note how Interview2 overlaps the bottom of Interview. 

### Current behavior
Events overlap
![image](https://user-images.githubusercontent.com/6614183/64245672-0a437780-cf14-11e9-84e4-92444635c58c.png)

### Expected/desired behavior
Events are positioned correctly, without overlapping. 

### Environment
* **Kendo UI version:** 2019.2.619
* **jQuery version:** x.y
* **Browser:** [all]

**Comment by veselints** [comment]
As a temporary workaround, use the following [dataBound](http://docs.telerik.com/kendo-ui/api/javascript/ui/scheduler#events-dataBound) event handler:

```` JavaScript
dataBound: function(e) {
  var allDayEvents = $('.k-scheduler-header-wrap .k-event');
  for(var i = 0; i < allDayEvents.length; i += 1) {
    var event = $(allDayEvents[i]);
    var initialString = event.css('top');
    var numberTop = Number(initialString.substring(0, initialString.length - 2));
    event.css('top', numberTop + i * 6 + 'px');
  }
},
````

**Comment by veselints** [comment]
Also reported in Ticket ID: 1102829

**Comment by magdalenaan** [comment]
The issue is not resolved with v2022.3.913
